src/TesApi.Web/AzureProxy.cs

```diff
@@ -57,34 +57,38 @@ public class AzureProxy : IAzureProxy
         private readonly string azureOfferDurableId;
         private readonly string batchResourceGroupName;
         private readonly string batchAccountName;
-
+        private AzureEnvironment azEnv;
```

Comment:
this can be readonly since it's set in the constructor, right?

Reply:
Already fixed.
```csharp
        private readonly AzureEnvironment azEnv;
```